### PR TITLE
a few random mappings

### PR DESCRIPTION
**Pull request by pyrox0**
whoops I did it again

More mappings in command, loot, item, enchantment, and one commit of misc changes to a few small parts of the mappings.

### CODE REVIEW

File: mappings/net/minecraft/enchantment/EnchantmentHelper.mapping [hunk 1]
```diff
@@ -106,6 +107,8 @@ CLASS net/minecraft/unmapped/C_jakrppis net/minecraft/enchantment/EnchantmentHel
 		ARG 4 treasureAllowed
 	METHOD m_vonozfhq hasSoulSpeed (Lnet/minecraft/unmapped/C_usxaxydn;)Z
 		ARG 0 entity
+	METHOD m_wfidublc getEnchantmentLevel (Lnet/minecraft/unmapped/C_jxtrubuh;Lnet/minecraft/unmapped/C_usxaxydn;)I
+		ARG 0 enchatment
```
Comment (eristhea) [inline mappings/net/minecraft/enchantment/EnchantmentHelper.mapping]:
```suggestion
		ARG 0 enchantment
```
typo fix
also requires adding GUSTER to the minecraft dictionary to fix build

File: mappings/net/minecraft/entity/passive/WolfVariant.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_qgilydva net/minecraft/entity/passive/WolfVariant
```
Comment (eristhea) [inline mappings/net/minecraft/entity/passive/WolfVariant.mapping]: mapped by #569

**Review by eristhea — APPROVED** [review]
looks great!

File: mappings/net/minecraft/entity/passive/WolfVariant.mapping (hunk 2 above)
Comment (pyrox0) [reply mappings/net/minecraft/entity/passive/WolfVariant.mapping]: Removed from this PR!

File: mappings/net/minecraft/item/ItemGroups.mapping [hunk 3]
```diff
@@ -29,6 +29,7 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 		ARG 0 collector
 		ARG 1 enchantmentLookup
 		ARG 2 targets
+		ARG 4 featureFlagBits
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/ItemGroups.mapping]:
```suggestion
		ARG 4 featureFlags
```
brief, and matches the already-mapped `tryReloadEntries` param

File: mappings/net/minecraft/item/ItemGroups.mapping [hunk 4]
```diff
@@ -39,6 +40,7 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 	METHOD m_jjegmdcl generateMaxEnchantedBookEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo;Ljava/util/Set;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;Lnet/minecraft/unmapped/C_czxxrbcp;)V
 		ARG 1 enchantmentLookup
 		ARG 2 targets
+		ARG 4 featureFlagBits
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/ItemGroups.mapping]:
```suggestion
		ARG 4 featureFlags
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]
tiny nitpick

File: mappings/net/minecraft/item/ItemGroups.mapping (hunk 4 above)
Comment (pyrox0) [reply mappings/net/minecraft/item/ItemGroups.mapping]: Fixed this for all 3 mappings that I set this for!

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**